urdf_parser/src/joint.cpp

```diff
@@ -639,46 +642,42 @@ bool exportPose(Pose &pose, TiXmlElement* xml);
 
 bool exportJointDynamics(JointDynamics &jd, TiXmlElement* xml)
 {
-  TiXmlElement *dynamics_xml = new TiXmlElement("dynamics");
-  dynamics_xml->SetAttribute("damping", urdf_export_helpers::values2str(jd.damping) );
-  dynamics_xml->SetAttribute("friction", urdf_export_helpers::values2str(jd.friction) );
-  xml->LinkEndChild(dynamics_xml);
+  auto * dynamics_xml = xmlAppendChild(xml, "dynamics");
+  dynamics_xml->SetAttribute("damping", jd.damping );
+  dynamics_xml->SetAttribute("friction", jd.friction );
   return true;
 }
 
 bool exportJointLimits(JointLimits &jl, TiXmlElement* xml)
 {
-  TiXmlElement *limit_xml = new TiXmlElement("limit");
-  limit_xml->SetAttribute("effort", urdf_export_helpers::values2str(jl.effort) );
-  limit_xml->SetAttribute("velocity", urdf_export_helpers::values2str(jl.velocity) );
-  limit_xml->SetAttribute("lower", urdf_export_helpers::values2str(jl.lower) );
-  limit_xml->SetAttribute("upper", urdf_export_helpers::values2str(jl.upper) );
-  xml->LinkEndChild(limit_xml);
+  auto * limit_xml = xmlAppendChild(xml, "limit");
+  limit_xml->SetAttribute("effort", jl.effort);
+  limit_xml->SetAttribute("velocity", jl.velocity );
```

Comment:
Nit: no space before the closing parentheses.  There are more instances of this below.